### PR TITLE
Adds grayscale to occupancy grid layer.

### PR DESCRIPTION
**Pull request by pedrolf**
Adds grayscale to occupancy grid layer.

@SirVer  ptal

### CODE REVIEW

File: android_15/src/org/ros/android/view/visualization/layer/OccupancyGridLayer.java [hunk 1]
```diff
@@ -216,11 +216,13 @@ private void update(nav_msgs.OccupancyGrid message) {
       if (pixel == -1) {
         tiles.get(tileIndex).writeInt(COLOR_UNKNOWN);
       } else {
-        if (pixel < 50) {
+        final int value = 250 - 2 * pixel;
+        tiles.get(tileIndex).writeInt(0xff000000 | (value << 16) | (value << 8) | value);
+        /*if (pixel < 50) {
```
Comment (SirVer) [inline android_15/src/org/ros/android/view/visualization/layer/OccupancyGridLayer.java]: remove the commented code.

File: android_15/src/org/ros/android/view/visualization/layer/OccupancyGridLayer.java [hunk 2]
```diff
@@ -216,11 +216,13 @@ private void update(nav_msgs.OccupancyGrid message) {
       if (pixel == -1) {
         tiles.get(tileIndex).writeInt(COLOR_UNKNOWN);
       } else {
-        if (pixel < 50) {
+        final int value = 250 - 2 * pixel;
```
Comment (SirVer) [inline android_15/src/org/ros/android/view/visualization/layer/OccupancyGridLayer.java:211]: Add a comment why this will not lead to invalid 'value's.

**Comment by SirVer** [comment]
Couple of nits.

**Comment by pedrolf** [comment]
@SirVer ptal

**Pull request merged**